src/js/index.js

```diff
@@ -1,140 +1,221 @@
+                <h3 class="setting-header">Account Management</h3>
+                <button class="account-management-btn">Change Password</button>
+                <button class="account-management-btn" id="uploader">Upload Resume</button>
+                <button class="account-management-btn">Log Out</button>
+            </div>
+        </div>
+    `;
+    
+    elements.container.insertAdjacentHTML('beforeend',AccountSettingsSetUp);
+
+    if(state.user.isSenior){
+        document.querySelector("#senior-btn").childNodes[1].checked = true;
+        document.querySelector("#junior-btn").childNodes[1].checked = false;
+    }
+}
+
+
+
+
+export const clearScreen = function(){
+   elements.container.innerHTML = "";
+};
+
+
+
+
+export const setUI = async function(state, user){
+    state.home = new Home(user.uid);
+    //console.log(user.uid);
+    state.user = user;
+    state.user.isSenior = false;
+    state.opp = new Opp("Ishmael");
+    state.messages = new Messages(user.uid);
+    state.image = "../images/kerwin.jpg";
+    state.home.collapsed = false;
+    state.menuCollapsed = false;
+    state.messages.firstRender = true;
+    //console.log(state.user.name);
+    tabSwitch(state, document.getElementById("default"));
+
+
+    //cleanse existing event listeners
+    const oldNavigator = document.querySelector('.tools');
+    const newNavigator = oldNavigator.cloneNode(true);
+    oldNavigator.parentNode.replaceChild(newNavigator, oldNavigator);
+    const oldCollapse = document.querySelector('.collapse');
+    const newCollapse = oldCollapse.cloneNode(true);
+    oldCollapse.parentNode.replaceChild(newCollapse, oldCollapse);
+
+    //3 main function tabs        
+    document.querySelector('.tools').addEventListener('click', e => {
+        e.preventDefault();
+        const tab = e.target.closest('.tab');
+        tabSwitch(state,tab);
+    });
+
+    //Account Settings
+    document.querySelector('#account-settings').addEventListener('click', e => {
+        e.preventDefault();
+        const tab = e.target.closest('.tab');
+
+        if (tab){
+            renderAccountSettings(state);
+            document.querySelector("#senior-btn").addEventListener('change', e => {
+                e.preventDefault();
+                const btn = e.target.closest('#senior-btn');
+
+                if (btn){
+                    document.querySelector("#opp-decision").textContent = 'Decision';
+                    state.user.isSenior = true;
+                    tabSwitch(state, document.getElementById("default"));
+                }
+            });
+
+            //???????????????????????????????????????????????????????????????
+            document.querySelector("#uploader").addEventListener('click', e => {
+                e.preventDefault();
+                const uploadBtn = e.target.closest('#uploader');
+
+                if(uploadBtn){
+                    console.log('test');
+                }
+            });
```

Comment:
What is this?
```suggestion
```

Reply:
This is the button created for Leo to test the storage functions in firebase.

Reply:
Why is this on this branch and not Leo's then?

Reply:
At the very least, you should name it appropriately, not `???????`
```suggestion

            //Testing button
            document.querySelector("#uploader").addEventListener('click', e => {
                e.preventDefault();
                const uploadBtn = e.target.closest('#uploader');

                if(uploadBtn){
                    console.log('test');
                }
            });
```